osu.Game.Tests/Beatmaps/IO/ImportBeatmapTest.cs

```diff
@@ -64,6 +66,9 @@ public void TestImportOverIPC()
                 ensureLoaded(osu);
 
                 waitForOrAssert(() => !File.Exists(temp), "Temporary still exists after IPC import", 5000);
+
+                host.Exit();
+                client.Exit();

```

Comment:
as discussed, unnecessary. apart from this i do believe this is how we want to do things (as long as `host.Run` is a blocking operation)

Reply:
the `client.Exit()` can still be removed here, right?